components/scream/src/share/io/tests/scorpio_tests.cpp

```diff
@@ -37,12 +44,12 @@ TEST_CASE("scorpio_interface_output", "") {
 
   int nerr = 0; // Set a record of the number of errors encountered.
   /* Create the set of SCORPIO output files and their respective dimensions and variables. */
-  int compid=0;  // For CIME based builds this will be the integer ID assigned to the atm by the component coupler.  For testing we simply set to 0
+  // int compid=0;  // For CIME based builds this will be the integer ID assigned to the atm by the component coupler.  For testing we simply set to 0
```

Comment:
Since these tests are not cime based, I think compid is not needed. If so, let's remove thise line.